Common/src/main/java/com/blamejared/crafttweaker/api/zencode/scriptrun/IScriptRun.java

```diff
@@ -0,0 +1,9 @@
+package com.blamejared.crafttweaker.api.zencode.scriptrun;
+
+public interface IScriptRun {
+    
+    IScriptRunInfo specificRunInfo();
```

Comment:
so there is also unspecific runInfo?

Reply:
Parse as "info for this specific run", as opposed to "info for the current run"